templates.dist/estate/default.php

```diff
@@ -61,6 +61,15 @@
 	while ( $currentEstate = $pEstatesClone->estateIterator() ) :
 		$marketingStatus = $currentEstate['vermarktungsstatus'];
 		unset($currentEstate['vermarktungsstatus']);
+		unset($currentEstate['referenz']);
```

Comment:
I'm curious: Why are we unsetting the values here? Are they conflicting with something?

Reply:
They don't conflict. It's not in use at the moment so I just hid it
Do you want to display it or use it in any way?
![image](https://user-images.githubusercontent.com/106214469/192955345-956fdcbe-1fb1-430e-a3d6-827f8620f805.png)

Reply:
Ah, ok! I forgot that they get rendered.

There is also the option of adding those fields to the [`dontEcho` array](https://github.com/onOffice-Web-Org/oo-wp-plugin/pull/324/files/79ce57d0bf507643e66912f02d29e1207a44ca16#diff-fe5dbe760501e4e626f9343c784248ce823bead31e4209da309b956045454e9cR30). ;)

Reply:
I added it you can review. :)